### ISSUE TITLE
[feat] Get system accent color on macOS

### ISSUE THREAD

**Issue by aurorarissime** [tauri-apps/tauri#8590]
### Describe the problem

Currently you can't use system accent colors on Mac because of wkwebview.

### Describe the solution you'd like

<img width="632" alt="Capture d’écran 2024-01-11 à 15 45 45" src="https://github.com/tauri-apps/tauri/assets/121387740/6a2a1cd9-c205-4dab-8ac6-0028b48db30c">

### Additional context

If needed : https://stackoverflow.com/questions/62797493/how-can-i-use-the-macos-system-accent-color-in-css

**Comment by aurorarissime** [comment]
https://canary.discord.com/channels/616186924390023171/1281144940675727360!!!
Can this be implemented maybe? Or since it's now a plugin it won't be officially added.

**Comment by FabianLars** [comment]
> Or since it's now a plugin it won't be officially added.

No, it generally should even raise the chances because then it can get tested up front a bit etc. Nonetheless this is really low prio for our small team so if there's nobody willing to contribute it will stay a plugin for a while :)
